buyer/issue/README.md

```diff
@@ -86,7 +87,8 @@ The `supplierAccountNumber` should be set on forehand in the Tradecloud connecti
 
 * `lines`: a purchase order contains one or multiple lines
 * `line`: a purchase order line which contains at least the position, item and delivery schedule. It is structured as a JSON element in the `lines` JSON array. 
-* `position`: the line position within the purchase order
+* `position`: the required line position identifier within the purchase order
+* `row`: the optional row label for this position. Only use a row when there is a distinction between position and row in your ERP system. The row will only be displayed in the portal.
```

Comment:
"The row will only be displayed in the portal." could maybe confuse people, they could think only the row is shown and position not. 

Maybe this is better: "When "position", as well as "row", are filled, only the "row" will be displayed in the portal."

Reply:
- The `position` will always be displayed as it is the identifier, the `row` will be displayed additionally
- I will replace "The row will only be displayed in the portal" with "Do NOT user row as identifier".